hoomd/md/pair/pair.py

```diff
@@ -298,8 +298,8 @@ class ExpandedLJ(Pair):
                \right)^{6} \right]
 
     Note:
-        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set
-        `hoomd.md.pair.Pair.r_cut` to ``r_cut_unshifted + delta``.
+        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set the
+        `hoomd.md.pair.Pair.r_cut` to ``r_cut(for delta=0) + delta``.
 
```

Comment:
This note may be more useful as an entry in the v3 migration guide rather than in the class docstring.

Reply:
Agree with this note.

Reply:
@cbkerr Shall we move it to the migration guide then?